settings.php

```diff
@@ -49,3 +49,15 @@
 $student = reset($student);
 $general->add(new admin_setting_configselect('mod_scormremote/roleid',
     get_string('defaultrole', 'role'), '', $student->id ?? null, $options));
+
+$choices = [
```

Comment:
should this have a set of fixed keys mapping to their lang string? We should never store a lang string as the actual admin setting value or it will break when you swap languages

Reply:
It's an indexed array so it would always be 0, 1, 2 ..
I made it explicit anyway.